doc/whats-new.rst

```diff
@@ -106,6 +106,13 @@ Deprecations
 
 Enhancements
 ~~~~~~~~~~~~
+
+- Added the xarray equivalent of `pandas.Dataframe.combine_first` as an instance
```

Comment:
just a note -- this will need to go in a new section for 0.9.1, since 0.9.0 will probably be released first.